### ISSUE TITLE
Mismatched types, expected `std::string::String`, found `&str`

### ISSUE THREAD

**Issue by Roryl-c** [Aleph-Alpha/ts-rs#81]
```rs
#[derive(Serialize, TS)]
#[ts(export)]
struct GetMeResponse(#[ts(type = "CurrentUser")] CurrentUser);
```

`CurrentUser` is a struct from a library that does not `derive(TS)`. I really just want to export that struct since I'm returning it directly as part of my API, but I'm getting this error when I try to do so:

```
mismatched types
expected `std::string::String` because of return type rustc(E0308)
rest.rs(19, 23): try using a conversion method: `.to_string()
```

The suggested conversion method obviously isn't valid in that position so I'm not sure how to resolve. Is this the correct way to export an external struct?

**Comment by NyxCode** [comment]
Thanks for the report. Fixed in 6.1.2 which was just released.

**Comment by Roryl-c** [comment]
Thanks for the quick fix!
